docker/deployment/scripts/kubeSetup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+MONGO=db-d8fd84c89-vjs7j
+APP=app-7c76db448d-sq5st
```

Comment:
`kubectl exec -it deploy/app --`

would always work, no need to manually enter the instance name

Reply:
```suggestion
APP=deploy/app
```

Reply:
I ended up getting the pod name programmatically since "deploy/all" works for the `exec` command but not the `cp` command.  Go figure.

Reply:
I am now happily exec'ing into a container using your suggest syntax of `kubectl exec -it deploy/app -- bash` so thank you for teaching me that.